dbt/tests/adapter/unit_testing/test_case_insensitivity.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from dbt.tests.util import run_dbt
+from dbt.tests.fixtures.project import *
```

Comment:
I would like to avoid using `from <module> import *` in general. It's less likely to cause an issue here since there are so few imports, but I'd still rather avoid it. Also, I don't see why this is needed here. The fixtures should already be registered via `tests/functional/conftest.py` or `tests/conftest.py`.